### DISCUSSION TITLE
ScriptReference/CharacterController.Move

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#83] [Notes]
# ScriptReference/CharacterController.Move

https://docs.unity3d.com/ScriptReference/CharacterController.Move.html

**Comment by vertxxyz** [comment]
If you are looking to teleport a CharacterController you need to disable it first:
```csharp
controller.enabled = false;
controller.transform.position = target.position;
controller.enabled = true;
```
If you are still having issues teleporting, call [`Physics.SyncTransforms`](https://docs.unity3d.com/ScriptReference/Physics.SyncTransforms.html) to apply Transform changes after you teleport the controller.
